### PR TITLE
return owner for files on windows

### PR DESCRIPTION
**Pull request by chris-rock**
This PR implements the owner methods for file/directory on Windows platforms. Once this is done, you can read our the owner via InSpec:

```
  describe directory('C:/opscode/chef') do
    its('owner') { should cmp 'NT AUTHORITY\SYSTEM'}
  end
  describe directory('C:/opscode/chef') do
    it { should be_owned_by 'NT AUTHORITY\SYSTEM' }
  end
  describe directory('C:/notthere') do
    its('owner') { should eq nil}
  end
```

### CODE REVIEW

File: lib/train/extras/file_windows.rb [hunk 1]
```diff
@@ -51,6 +51,26 @@ def mounted
...
     end
 
+    # PS C:\> Get-Acl .\test | select owner,group,audit | ConvertTo-Json
+    # {
+    #     "Owner":  "BUILTIN\\Administrators",
+    #     "Group":  "DEFAULT-CHEF-WI\\None",
+    # }
+    def owner
+      cmd = @backend.run_command(
+        "Get-Acl '#{@spath}' | select owner, group | ConvertTo-Json"
```
Comment (ksubrama) [inline lib/train/extras/file_windows.rb]: If you just want the owner, you can just return stdout from `Get-Acl '#{@spath}' | select -expand Owner`

Comment (chris-rock) [inline lib/train/extras/file_windows.rb]: Uh, nice, lets use that instead.

**Comment by chris-rock** [comment]
Thanks @ksubrama for the tip, that made the implementation much easier

**Comment by chris-rock** [comment]
I am going to ignore the travis test for fedora, since this is combination of docker and travis.

**Pull request merged**